R/utils.r

```diff
@@ -123,7 +123,7 @@ first_upper <- function(x) {
 }
 
 download <- function(path, url, ...) {
-  request <- httr::GET(url, ...)
+  request <- GET(url, ...)
```

Comment:
Yep sorry, copied this code from an old blame before we were using `httr::` and I missed fixing this line.